cdp-sdk-go/cdp/credentials.go

```diff
@@ -140,7 +140,7 @@ func (p *ChainCredentialsProvider) GetCredentials() (*Credentials, error) {
 // maps to a set of key value pairs.
 func rawParseConfigFile(path string) (map[string]map[string]string, error) {
 	properties := make(map[string]map[string]string)
-	cfg, err := ini.InsensitiveLoad(path)
+	cfg, err := ini.Load(path)
```

Comment:
Can we mimic the behavior from the Python CDP CLI ConfigParser: https://docs.python.org/3/library/configparser.html#customizing-parser-behaviour which uses case sensitive section names, but case insensitive keys:

```suggestion
		cfg, err := ini.LoadSources(
			ini.LoadOptions{
				InsensitiveKeys: true,
			},
		path)
```

Reply:
Thanks @enis - I updated to your example.